### PR TITLE
rrdtool: 1.7.0 -> 1.7.0.2017-06-11 (bugfix in upstream)

### PR DESCRIPTION
**Pull request by volth**
###### Motivation for this change

```1.7.0``` is unusable because of bug https://github.com/NixOS/nixpkgs/issues/26780

Fixes https://github.com/NixOS/nixpkgs/issues/26780

cc: @pSub 
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/rrdtool/default.nix [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  name = "rrdtool-1.7.0.2017-06-23";
 
-  src = fetchurl {
-    url = "http://oss.oetiker.ch/rrdtool/pub/${name}.tar.gz";
-    sha256 = "0ssjqpa0dwwzbylc0drmlbq922qcw8crffc0rpr805xr6n4k8zgr";
+  src = fetchFromGitHub {
+    owner = "oetiker";
+    repo = "rrdtool-1.x";
+    rev = "e0eb6257ad9b150636ef0ccc3797b71e6a3e822a";
+    sha256 = "0szjnvwailsianlhxvvgzr97d78nvkkid3imcfvawy5w4x6bvasv";
```
Comment (Mic92) [inline pkgs/tools/misc/rrdtool/default.nix]: Is it fixed in a certain commit? We usually use fetchpatch to download just the commit.

Comment (volth) [inline pkgs/tools/misc/rrdtool/default.nix]: There are two commits.

Comment (volth) [inline pkgs/tools/misc/rrdtool/default.nix]: https://github.com/oetiker/rrdtool-1.x/commit/e1bddaf532ad29cf660318955875b1183000d850

**Comment by volth** [comment]
@Mic92 [they found and fix weird bugs every week](https://github.com/oetiker/rrdtool-1.x/commits/master) and do not release minor versions after that. Shouldn't we better follow github version instead of using 1.7.0 + patches from github?

**Pull request closed without merging**